x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/create_datasource_page/services/validate_datasource.test.ts

```diff
@@ -274,7 +314,18 @@ describe('Ingest Manager - validateDatasource()', () => {
         },
       },
       'with-disabled-streams': {
-        streams: { 'with-disabled-streams-disabled': { vars: { 'var-name': null } } },
+        streams: {
+          'with-disabled-streams-disabled': {
+            vars: { 'var-name': null },
+          },
+          'with-disabled-streams-disabled2': {},
```

Comment:
Is this an intentional addition, and if yes, can it have a more descriptive name than just the `2` added at the end?